projects/netgrif-components-core/src/lib/data-fields/user-list-field/models/user-list-value.ts

```diff
@@ -36,6 +36,11 @@ export class UserListValue {
         return Array.from(this._userValues.values()).pop();
     }
 
+    public toString(): string {
+        const userNames = Array.from(this._userValues.values()).reduce((acc, curr) => acc + curr.fullName + ',', '')
```

Comment:
`Array.join()` would be more readable I think
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join